src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +100,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(new CandidateValidator())
```

Comment:
```suggestion
                .filter(CandidateValidator::new)
```

You didn't fix the mentors comment

Reply:
@Elena-Bruyako mentor said that the reference to the constructor does not work with predicates and I can leave my variant